server/github.go

```diff
@@ -16,9 +16,10 @@ import (
 const gitHubAPI = "https://api.github.com"
 
 type GitHubCommit struct {
-	Author *models.User `json:"author"`
-	Sha    string       `json:"sha"`
-	Commit struct {
+	Author   *models.User `json:"author"`
+	Sha      string       `json:"sha"`
+	HTML_URL string       `json:"html_url"`
```

Comment:
How about `HtmlURL` here? That would match the [naming here](https://github.com/Wind0r/applikatoni/blob/f1f5cef287d5989ae2e1b6a2e6df68466f853ffd/server/github.go#L48-L53).